master/buildbot/test/unit/test_steps_http.py

```diff
@@ -99,15 +99,19 @@ def test_404(self):
 
     def test_POST(self):
         url = self.getURL("POST")
-        content = "\n<html>\n" \
-                  "  <head><title>405 - Method Not Allowed</title></head>\n" \
-                  "  <body>\n    <h1>Method Not Allowed</h1>\n    <p>Your browser " \
-                  "approached me (at /POST) with the method \"POST\".  I only allow the " \
-                  "methods HEAD, GET here.</p>\n  </body>\n</html>\n"
+        #
```

Comment:
I already disable that test in one of my PR I believe we should fix the thing properly. probably hardcoding the whole log is not a good idea, but looking at some of its content looks better